tools/update-github-project-board/index.ts

```diff
@@ -0,0 +1,277 @@
+import { Octokit } from "@octokit/rest";
+import { graphql } from "@octokit/graphql";
+
+const ORG = process.env.ORGANIZATION || "hashicorp";
+const PROJECT_ID = parseInt(process.env.PROJECT_NUMBER || "125", 10);
+type Awaited<T> = T extends PromiseLike<infer U> ? U : T;
+type Issue = Awaited<ReturnType<typeof octokit.issues.list>>["data"][0];
+
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN,
+});
+
+const gql = graphql.defaults({
+  headers: {
+    authorization: `token ${process.env.GITHUB_TOKEN}`,
+  },
+});
+
+async function allRepos() {
```

Comment:
nit: This is called `allRepos` while another function below is named `getCurrentProjectItemIds` 😁 (The consistency pet peeve of mine we recently talked about 😂)